.github/copilot-instructions.md

```
@@ -0,0 +1,74 @@
+# hass-opnsense Home Assistant Integration Copilot Instructions
+
+## General Guidelines
+- Follow Home Assistant's developer documentation: https://developers.home-assistant.io/docs/.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)